playwright/async_api/_context_manager.py

```diff
@@ -28,23 +28,28 @@ def __init__(self) -> None:
 
     async def __aenter__(self) -> AsyncPlaywright:
         loop = asyncio.get_running_loop()
+        on_ready_future: asyncio.Future = asyncio.Future()
         self._connection = Connection(
             None,
             create_remote_object,
-            PipeTransport(loop, compute_driver_executable()),
+            PipeTransport(
+                loop,
+                compute_driver_executable(),
+                lambda: on_ready_future.set_result(None),
+            ),
         )
         self._connection._loop = loop
         loop.create_task(self._connection.run())
-        playwright_future = loop.create_task(
-            self._connection.wait_for_object_with_known_name("Playwright")
-        )
+        await on_ready_future
```

Comment:
Let's plumb it through connection properly: 
```
playwright = await self._connection.connect()
```

Reply:
This will hurt us in the sync implementation I guess.